$NetBSD: patch-aa,v 1.2 2000/08/29 20:48:15 jlam Exp $

Until the sources catch up to the current Perl API, we need PERL_POLLUTE
to look like an older perl.

--- Makefile.PL.orig	Mon Aug  3 12:59:14 1998
+++ Makefile.PL
@@ -32,17 +32,17 @@
    'PREREQ_PM'    => { Curses     => 0,
                      },
    'LIBS'         => "-L$Config{ldflags} -lwais",
-   'INC'          => "-DWAIS_USES_STDIO -I$Config{cppflags}",
+   'INC'          => "-DWAIS_USES_STDIO $Config{cppflags}",
    'NAME'         => 'Wais',
    'VERSION_FROM' => 'lib/Wais.pm',
-   'DEFINE'       => '-DTRACE=0',
+   'DEFINE'       => '-DTRACE=0 -DPERL_POLLUTE=1',
    'OBJECT'       => "Wais.o HTWAIS.o dictionary.o",
    'dist'         => { SUFFIX   => 'gz',
                        COMPRESS => 'gzip -9f',
                        PREOP    => 'pod2text lib/Wais.pm > README; pod2text install > INSTALL',
                      },
    'MAP_TARGET'   => 'waisperl',
-   'EXE_FILES'    => [ qw(script/inspect script/makedb) ],
+   'EXE_FILES'    => [ qw(script/inspect) ],
   );
 
 
@@ -55,7 +55,7 @@
   my $self = shift;
   
   qq[
-install :: register
+#install :: register
 
 register ::
 \t] . $self->{NOECHO} . q[$(PERL) -I$(PERL_ARCHLIB) -I$(PERL_LIB) \
